plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/Completer/Paths.hs

```diff
@@ -64,7 +85,7 @@ pathCompletionInfoFromCabalPrefixInfo srcDir prefInfo =
     }
   where
     prefix = T.unpack $ completionPrefix prefInfo
-    (queryDirectory', pathSegment') = Posix.splitFileName prefix
+    (queryDirectory', pathSegment') = splitFileNameNoTrailingSlash prefix             
```

Comment:
```suggestion
    (queryDirectory', pathSegment') = splitFileNameNoTrailingSlash prefix
```

Reply:
Please configure your editor to trim trailing whitespace https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut/53663494#53663494